File: src/main/java/cn/nm/lms/carpetlmsaddition/rules/spongerecipe/SpongeRecipe.java
/*
 * Copyright (C) 2025  Carpet-LMS-Addition contributors
 * https://github.com/Citrus-Union/Carpet-LMS-Addition

 * Carpet LMS Addition is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, version 3 of the License.

 * Carpet LMS Addition is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.

 * You should have received a copy of the GNU General Public License
 * along with Carpet LMS Addition.  If not, see <https://www.gnu.org/licenses/>.
 */
package cn.nm.lms.carpetlmsaddition.rules.spongerecipe;

import carpet.api.settings.Rule;
import carpet.api.settings.RuleCategory;
import cn.nm.lms.carpetlmsaddition.rules.LMSRuleCategory;

public final class SpongeRecipe {
  @Rule(categories = {LMSRuleCategory.LMS, RuleCategory.SURVIVAL})
  public static boolean spongeRecipe = false;

  private SpongeRecipe() {}
}

File: src/main/kotlin/cn/nm/lms/carpetlmsaddition/rules/spongerecipe/SpongeRecipe.kt
package cn.nm.lms.carpetlmsaddition.rules.spongerecipe

import carpet.api.settings.Rule
import carpet.api.settings.RuleCategory
import cn.nm.lms.carpetlmsaddition.rules.LMSRuleCategory

object SpongeRecipe {
    @Rule(
        categories = [LMSRuleCategory.LMS, RuleCategory.SURVIVAL],
    )
    @JvmField
    var spongeRecipe: Boolean = false
}
